ci/builders/linux_web_engine_framework.json

```diff
@@ -0,0 +1,389 @@
+                "--no-goma"
+            ],
+            "name": "ci/android_debug",
+            "description": "Produces debug mode artifacts to target 32-bit arm Android from a Linux host.",
+            "ninja": {
+                "config": "ci/android_debug",
+                "targets": [
+                    "flutter",
+                    "flutter/sky/dist:zip_old_location",
+                    "flutter/shell/platform/android:embedding_jars",
+                    "flutter/shell/platform/android:abi_jars"
+                ]
+            }
+        },
+        {
+            "archives": [
+                {
+                    "name": "ci/host_release",
+                    "base_path": "out/ci/host_release/zip_archives/",
+                    "type": "gcs",
+                    "include_paths": [
+                        "out/ci/host_release/zip_archives/flutter_patched_sdk_product.zip"
+                    ]
+                }
+            ],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "dependencies": [
+                {
+                    "dependency": "goldctl",
+                    "version": "git_revision:720a542f6fe4f92922c3b8f0fdcc4d2ac6bb83cd"
+                }
+            ],
+            "gclient_variables": {
+                "download_android_deps": false,
+                "use_rbe": true
+            },
+            "gn": [
+                "--target-dir",
+                "ci/host_release",
+                "--runtime-mode",
+                "release",
+                "--prebuilt-dart-sdk",
+                "--build-embedder-examples",
+                "--rbe",
+                "--no-goma"
+            ],
+            "name": "ci/host_release",
+            "description": "Produces release mode Linux host-side tooling and builds host-side unit tests and benchmarks for Linux.",
+            "ninja": {
+                "config": "ci/host_release",
+                "targets": [
+                    "flutter/build/dart:copy_dart_sdk",
+                    "flutter/tools/path_ops",
+                    "flutter/build/archives:flutter_patched_sdk"
+                ]
+            }
+        }
+    ],
+    "tests": [
+        {
+            "name": "web_tests-0",
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "dependencies": [
+                "ci/android_debug",
+                "ci/host_debug",
+                "ci/host_release",
+                "web_tests/artifacts"
+            ],
+            "test_dependencies": [
+                {
+                    "dependency": "chrome_and_driver",
+                    "version": "version:119.0.6045.9"
+                },
+                {
+                    "dependency": "curl",
+                    "version": "version:7.64.0"
+                }
+            ],
+	    "shard": "web_tests",
```

Comment:
I think there's too much magic going on here. Does this really encode all of these?
1. Checking out flutter/flutter at some hash defined elsewhere
2. Finding the "web_tests" test. (How does it find it? How does one make some *other* test visible here?)
3. Knowledge of the specific way that this set of framework tests currently do sharding

Instead, I think this build config should be made a lot more explicit, and I think the way to do that is by having the logic live in Dart code in the engine repo instead of python code in the recipes repo. That is, the logic for choosing the framework commit, checking out the framework, and launching the tests should live in the engine repo. Then, we can simply use the features that already exist in engine_v2 without adding more magic fields to the build config spec.